PackageReadme.md

```diff
@@ -1,9 +1,21 @@
-[Steeltoe](https://steeltoe.io/) provides building blocks for development of ASP.NET Core Web APIs and microservices that integrate with [Java Spring](https://spring.io/) and [Spring Boot](https://spring.io/projects/spring-boot) environments, as well as [Cloud Foundry](https://www.cloudfoundry.org/) and [Kubernetes](https://kubernetes.io/) ([Tanzu](https://tanzu.vmware.com/tanzu)).
+# Steeltoe&nbsp;[![Build Status](https://dev.azure.com/SteeltoeOSS/Steeltoe/_apis/build/status/Steeltoe.All?branchName=main)](https://dev.azure.com/SteeltoeOSS/Steeltoe/_build/latest?definitionId=4&branchName=main)&nbsp;[![NuGet Version](https://img.shields.io/nuget/v/Steeltoe.Common.svg?style=flat)](https://www.nuget.org/packages/Steeltoe.Common/)&nbsp;[![Stack Overflow](https://img.shields.io/badge/stack%20overflow-steeltoe-orange.svg)](http://stackoverflow.com/questions/tagged/steeltoe)
```

Comment:
The build status badge doesn't belong here. This README is supposed to be a point-in-time document. It should not reference "the latest build", which _never_ applies to the specific package version this document belongs to.

The Nuget badge is redundant, anyone reading this is already looking at this package on nuget.org.

Reply:
Sorry for the confusion, I added the suggestion to eliminate some extra push/pull